index.html

```diff
@@ -9,15 +9,21 @@
   </head>
 
   <body>
+    <!--Description of Website-->
+    <h1>Anderson University Spring 2023 Course Catalog</h1>
+    <h3>Course Search Based on Start Time</h3>
+    <input type="text" id="timeSearch" placeholder="ex. 09:00 AM" />
+    <button id="search-button">Search</button>
     <table>
       <thead>
         <tr>
-          <td>Course</td>
```

Comment:
do you want to delete the heading... why not just have your td underneath the heading instead of replacing it

Reply:
Yes we can delete the heading, it's not necessary

Reply:
Actually never mind, I just looked at it... it looks good

Reply:
Okay works for me

Reply:
Just make sure you pull master and merge it into your branch